pkg/pod/status_test.go

```diff
@@ -971,3 +867,32 @@ func TestMarkStatusSuccess(t *testing.T) {
 		t.Errorf("Unexpected status: %s", diff.PrintWantGot(d))
 	}
 }
+
+func statusRunning() duckv1beta1.Status {
```

Comment:
thank you @adshmh for this refactoring 🙏 would like to see using [t.Helper()](https://blog.golang.org/go1.9) here so that it reports accurate line number in case of failure. And similarly, rest of the three functions statusFailure, statusSuccess, and statusPending.

Reply:
Thank you for the review. Fixed.